src/app/social-media/social-media-input/social-media-input.component.html

```diff
@@ -0,0 +1,57 @@
+<nav id="social-media-list-input-nav" class="navbar fixed-top navbar-expand-lg bg-primary navbar-light pt-0 pb-0">
+    <div class="container-fluid">
+        <div class="navbar-brand title mr-auto">
+            <h4>Update Social Media List</h4>
+        </div>
+        <ul class="navbar-nav navbar-right">
+            <li class="nav-item btn-link"><a href="#" (click.preventDefault)="addSocialMediaPost()" class="btn btn-link">Add</a></li>
+            <li class="nav-item btn-link"><a href="#" (click.preventDefault)="backToSocialMediaList()" class="btn btn-link">Back To List</a></li>
+        </ul>
+    </div>
+</nav>
+
+<div id="admin-social-media-list-input-form" class="container-fluid pl-0 pr-0 pt-5">
+<div id="social-media-input-list" class="card-columns">
+  <div *ngFor="let item of socialmedia" [ngSwitch]="item.mediaType" class="social-media-post">
+    <div class="card" *ngSwitchCase="'Twitter'">
+      <div class="btn-toolbar pull-right" role="toolbar" aria-label="Toolbar with button groups">
+        <div class="btn-group mr-2" role="group" aria-label="First group">
+          <button class="btn btn-link" type="submit" (click.preventDefault)="pinSocialMediaPost(item)"><i class="fa fa-thumb-tack fa-lg" [ngStyle]="{'color': getPinColor(item.sortOrder)}"></i></button>
```

Comment:
the click.preventDefault is only needed on <a> elements to prevent the page from navigating to the location. This button won't have a default so there is nothing to prevent

Reply:
oh nevermind the button is a type=submit. Does it need to be?

Reply:
@AlessiaYChen I would like to see all of the buttons of type=button without (click.preventDefault) and instead using just regular (click). This one here should probably be of type=button instead of submit as well